src/components/common/MovieCard.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="card">
+    <CardTitle :text="movie.title" />
+    <CardSubtitle :text="movie.length" />
+    <div class="card__image">
+      <img :src="imgUrl" />
+    </div>
+    <div class="card__tags">
+      <CustomChip v-for="tag of movie.tags" :key="tag" :text="tag"></CustomChip>
+    </div>
+  </div>
+</template>
+<script>
+import { CardTitle, CardSubtitle, CustomChip } from "./index";
+export default {
+  components: { CardTitle, CardSubtitle, CustomChip },
+  props: {
+    movie: { type: Object, required: true },
+  },
+  setup(props) {
+    const imgUrl = new URL(
+      `/src/assets/images/${props.movie.image}`,
+      import.meta.url
+    ).href;
```

Comment:
Likewise in the previous situation, it's a good practice to make anything depending on the props a computed (which you can import from vue) 💡